Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -71,6 +73,20 @@ final class ChatViewController: SLKTextViewController {
         didSet {
             guard !subscription.isInvalidated else { return }
 
+            subscriptionToken?.invalidate()
```

Comment:
Needs to invalidate the token here too: https://github.com/RocketChat/Rocket.Chat.iOS/blob/9ed56d60e288823a96f920bf68754f49361d0e10/Rocket.Chat/Controllers/Chat/MainChatViewController.swift#L81-L96